nucliadb_core/src/metrics/metric/tokio_runtime.rs

```diff
@@ -0,0 +1,444 @@
+// Histogram metric
+const BUCKETS: [f64; 15] = [
+    0.000010, 0.000025, 0.000050, 0.000100, 0.000250, 0.000500, 0.001, 0.002, 0.005, 0.010, 0.100,
+    0.250, 0.500, 1.0, 5.0,
+];
+
+impl TokioRuntimeMetrics {
+    fn new(registry: &mut Registry) -> Self {
+        let workers_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_workers_count",
+            "The number of worker threads used by the runtime. This metric is static for a runtime",
+            workers_count.clone(),
+        );
+
+        let total_park_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_total_park_count",
+            "The number of times worker threads parked. [...]",
+            total_park_count.clone(),
+        );
+
+        let max_park_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_max_park_count",
+            "The maximum number of times any worker thread parked",
+            max_park_count.clone(),
+        );
+
+        let min_park_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_min_park_count",
+            "The minimum number of times any worker thread parked",
+            min_park_count.clone(),
+        );
+
+        let mean_poll_duration = Histogram::new(BUCKETS.iter().copied());
+        registry.register(
+            "nucliadb_node_mean_poll_duration",
+            "The average duration of a single invocation of poll on a task. [...]",
+            mean_poll_duration.clone(),
+        );
+
+        let mean_poll_duration_worker_min = Histogram::new(BUCKETS.iter().copied());
+        registry.register(
+            "nucliadb_node_mean_poll_duration_worker_min",
+            "The average duration of a single invocation of poll on a task oin the worker with \
+             the lowest volume. [...]",
+            mean_poll_duration_worker_min.clone(),
+        );
+
+        let mean_poll_duration_worker_max = Histogram::new(BUCKETS.iter().copied());
+        registry.register(
+            "nucliadb_node_mean_poll_duration_worker_max",
+            "The average duration of a single invocation of poll on a task on the worker with the \
+             highest value. [...]",
+            mean_poll_duration_worker_max.clone(),
+        );
+
+        // This metric must be explicitly enabled (and we don't)
+        // let poll_count_histogram = Histogram::new(BUCKETS.iter().copied());
+        // registry.register(
+        //     "nucliadb_node_poll_count_histogram",
+        //     "A histogram of task polls since the previous probe grouped by poll times. [...]",
+        //     poll_count_histogram.clone(),
+        // );
+
+        let total_noop_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_total_noop_count",
+            "The number of times worker threads unparked but performed no work before parking \
+             again. [...]",
+            total_noop_count.clone(),
+        );
```

Comment:
Shouldn't this be a  `Counter`? Based on the description it seems like is something that can only increase right?